catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -740,7 +739,16 @@ public Response updateDocument(
 
         if (createInfo == null) {
           UpdateRequest updateRequest =
-              new UpdateRequestImpl(id, generateMetacard(mimeType, id, message, transformerParam));
+              new UpdateRequestImpl(
```

Comment:
Creating a local variable for the second argument would improve readability.

Reply:
done, and added a check for a possible empty list